platformio.ini

```diff
@@ -1362,6 +1362,8 @@ lib_ignore    = NativeEthernet
 upload_speed  = 115200
 #upload_port   = marlinesp.local
 #board_build.flash_mode = qio
+# Please delete '#' of following line if you use FYSETC E4 board
+#board_build.partitions = default_16MB.csv
```

Comment:
Does this csv file come from the framework, or should it be included in this PR?

Reply:
It is in framework.